tools/jetty/etc/enabled-jetty-config

```diff
@@ -18,6 +18,7 @@ jetty-jaas.xml
 
 
 ## Main Server Config
+jetty-gzip.xml
```

Comment:
Was it tested? It was disabled because of problems it bring. Can this be disabled by default?